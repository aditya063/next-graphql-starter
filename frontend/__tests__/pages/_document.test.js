import React from 'react';
import { shallow } from 'enzyme';
import MyDocument from '../../pages/_document';

describe('_document', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<MyDocument {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('is defined', () => {
    expect(wrapper).toBeDefined();
  });
});
