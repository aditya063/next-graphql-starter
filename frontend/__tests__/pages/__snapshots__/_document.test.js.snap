// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_document matches snapshot 1`] = `
<html
  lang="en"
>
  <Head>
    <meta
      charSet="utf-8"
    />
    <meta
      content=""
      name="description"
    />
    <meta
      content="
 width=device-width,
 height=device-height,
 initial-scale=1,
 minimum-scale=1,
 user-scalable=0,
 shrink-to-fit=no"
      name="viewport"
    />
    <link
      href="/static/favicon.ico"
      rel="shortcut icon"
      type="image/x-icon"
    />
    <link
      as="font"
      crossOrigin="anonymous"
      href="/static/fonts/open-sans-v15-latin-regular.woff2"
      rel="preload"
      type="font/woff2"
    />
    <link
      as="font"
      crossOrigin="anonymous"
      href="/static/fonts/play-v10-latin-regular.woff2"
      rel="preload"
      type="font/woff2"
    />
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</html>
`;
